Documentation/metrics.md

```diff
@@ -46,3 +46,21 @@ Abnormally high fsync duration (`fsync_durations_microseconds`) indicates disk i
 | snapshot_save_total_durations_microseconds | The total latency distributions of save called by snapshot | Summary |
 
 Abnormally high snapshot duration (`snapshot_save_total_durations_microseconds`) indicates disk issues and might cause the cluster to be unstable.
+
+### rafthttp
+
+| Name                              | Description                                | Type    | Labels                     |
+|-----------------------------------|--------------------------------------------|---------|----------------------------|
+| message_sent_latency_microseconds | The latency distributions of messages sent | Summary | channel, msgType, remoteID |
+| message_sent_failed_total         | The total number of failed messages sent   | Summary | channel, msgType, remoteID |
+
+
+Abnormally high message duration (`message_sent_latency_microseconds`) indicates network issues and might cause the cluster to be unstable.
+
+An increase in message failures (`message_sent_failed_total`) indicates more severe network issues and might cause the cluster to be unstable.
+
+Label `channel` is the channel to send message. `message`, `msgapp` and `msgappv2` channels use HTTP streaming, while `pipeline` does HTTP request for each message.
+
+Label `msgType` is the type of raft message. The size of `MsgApp` is usually smaller than 1MB, and the size of `MsgSnap` equals to the size of snapshot file. The size of all other types of messages is less than 100B.
```

Comment:
why do we mention about message size? I think we need to mention the relationship between size and latency (we only record latency)